### PR TITLE
Task/11779 changes certificate overview screen

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Changes to health certificate overview screen 

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11779

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -2805,11 +2805,9 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 "HealthCertificate_Overview_title" = "Zertifikate";
 
-"HealthCertificate_Overview_add" = "Zertifikat hinzufügen";
-
 "HealthCertificate_Overview_EmptyTitle" = "Noch keine Zertifikate vorhanden";
 
-"HealthCertificate_Overview_EmptyDescription" = "Hier werden die gespeicherten digitalen Impfzertifikate, Genesenenzertifikate und Testzertifikate angezeigt.\n\nUm ein Zertifikat hinzuzufügen, scannen Sie bitte den dazugehörigen QR-Code.";
+"HealthCertificate_Overview_EmptyDescription" = "Hier werden die gespeicherten digitalen Impfzertifikate, Genesenenzertifikate und Testzertifikate angezeigt.\n\nScannen Sie den QR-Code des Zertifikats mit Hilfe des QR-Code Scanners in der unteren Navigationsleiste.";
```
Comment (CV113) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:2810]: A hyphen is missing
 WRONG: QR-Code Scanners 
 RIGHT: QR-Code-Scanners

**Review by CV113 — CHANGES_REQUESTED** [review]

**Review by CV113 — APPROVED** [review]
UA approved

**Review by KaiTeuber — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/HealthCertificateOverviewViewModel.swift [hunk 2]
```diff
@@ -62,12 +64,16 @@ class HealthCertificateOverviewViewModel {
 
 	func numberOfRows(in section: Int) -> Int {
 		switch Section(rawValue: section) {
-		case .createCertificate:
-			return 1
+		case .changeAdmissionScenarioStatusLabel:
+			return healthCertifiedPersons.isEmpty ? 0 : 1
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/HealthCertificateOverviewViewModel.swift:68]: this logic will be replaced by task 11811
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11811

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/__tests__/ChangeAdmissionScenarionCellModelTests.swift [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	func testGIVEN_CellModel_THEN_InitialStateIsAsExpected() {
+		// GIVEN
+		let cellViewModel = ChangeAdmissionScenarionCellModel()
+
+		// THEN
+		XCTAssertEqual(cellViewModel.textColorPublisher.value, UIColor.enaColor(for: .textPrimary1))
+		XCTAssertEqual(cellViewModel.iconImagePublisher.value, UIImage(named: "Icons_Admission_State"))
+		XCTAssertFalse(cellViewModel.isCustomAccessoryViewHiddenPublisher.value)
+	}
+
```
[the remark this answers is not in the document]
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/__tests__/ChangeAdmissionScenarionCellModelTests.swift:19]: Yes

**Review by pascal-brause — APPROVED** [review]

**Pull request merged**